### ISSUE TITLE
Allow suppression of specific errors?

### ISSUE THREAD

**Issue by StevenLangbroek** [stylelint/stylelint#3278]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I have a quite elaborate project (Next.js, Typescript, Ant Design) with a lot of abstractions in between "userland" and `stylelint` for example. Somewhere in all of this, duplicate inline disables are being generated. This leads to these errors:

```
"no-duplicate-selectors" has already been disabled
```

As far as I can tell there's no way to suppress these warning. Something like:

```json
// .stylelintrc
{
  "ignoreDuplicateDisables": true,
  "plugins": "selector-bem-pattern"
}
```

> Which rule, if any, is this issue related to?

None.

> What CSS is needed to reproduce this issue?

None.

> What stylelint configuration is needed to reproduce this issue?

None.

> Which version of stylelint are you using?

`9.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

3rd party `less` code

**Comment by evilebottnawi** [comment]
@StevenLangbroek it is don't break linting? Just warning?

**Comment by jeddy3** [comment]
This doesn't feel like a common feature as the recommendation is to run stylelint on your sources files, which are generally small in size and so overlapping disable commands should be flagged.  

Closing as part of issue triage. We can always revisit it if it gets a lot of thumbs up.
